scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -108,6 +108,8 @@ package object dialects {
     .withAllowGivenUsing(true)
     .withAllowExtensionMethods(true)
     .withAllowOpenClass(true)
+    .withAllowWhiteboxMacro(true)
```

Comment:
I have no preferences, let me know if you want it to be separate dialect and I will squash all settings.
It will be Dialect(a, b, c).withX(..).withY(..) - some parameters go with constructor, other are only exposed via `withX`

Reply:
Let's leave it as is.